spyder/widgets/helperwidgets.py

```diff
@@ -17,18 +17,22 @@
 from qtpy.QtCore import (
     QPoint, QRegExp, QSize, QSortFilterProxyModel, Qt, Signal)
 from qtpy.QtGui import (QAbstractTextDocumentLayout, QColor, QFontMetrics,
-                        QPainter, QRegExpValidator, QTextDocument, )
+                        QPainter, QRegExpValidator, QTextDocument, QImage)
+from qtpy.QtSvg import QSvgRenderer
```

Comment:
```suggestion
                        QPainter, QRegExpValidator, QTextDocument, QImage,
                        QPixmap)
from qtpy.QtSvg import QSvgRenderer
```

`QPixmap` can be imported here instead of doing it below.